### ISSUE TITLE
How to run

### ISSUE THREAD

**Issue by assaf007** [aemreunal/iBeaconServer#1]
Hello, how can I configure and run the project?
I test it like you said in the discription but it don't work.
Thanks.

**Comment by aemreunal** [comment]
Hello,
Thank you for your interest! I haven't ran this application for almost 2 years now and I'm unsure as to whether it even runs anymore. I'm, unfortunately, no longer am maintaining this project. However, if you can give me details of what you're trying to do, what's the part that's not working, etc., I can try to help you.

**Comment by assaf007** [comment]
Thanks for your reply. I have have ionic client which can detect iBeacons. I need server Back-End to manage this iBeacons. 
When I run "deploy.sh" I have nothing.
I don't see the "tomcatRunWar" in the project.
Maybe I miss something, do I need to install/configure something before this commands?
Thank you in advance.

**Comment by aemreunal** [comment]
What's the output of running `deploy.sh`?

**Comment by assaf007** [comment]
As you said I create new folder called "ibeaconserver" and copy all the files inside "Docker" folder to "ibeaconserver". When I run the deploy.sh I have the following output:

![output](https://cloud.githubusercontent.com/assets/18464681/25375763/1af373ba-29a2-11e7-8382-727ff2c1c5aa.png)

**Comment by aemreunal** [comment]
Well, it says it right there:

**Comment by aemreunal** [comment]
Well, it says it right there: 
> "In the default daemon configuration on Windows, the Docker client must be run elevated to connect. This error may also indicate that the docker daemon is not running."

Did you go through these steps?

**Comment by assaf007** [comment]
Yes, I have the same error, my docker machine is running (I see it in VirtualBox).

**Comment by aemreunal** [comment]
Is it running with elevated permissions?

**Comment by assaf007** [comment]
I'm on Windows 10, how can I run it with elevated permissions?
